src/actions/actions.ts

```diff
@@ -782,7 +782,7 @@ export class CommandSearchForwards extends BaseCommand {
   isMotion = true;
 
   public async exec(position: Position, vimState: VimState): Promise<VimState> {
-    vimState.searchState = new SearchState(SearchDirection.Forward, vimState.cursorPosition);
+    vimState.searchState = new SearchState(SearchDirection.Forward, vimState.cursorPosition, "", true);
```

Comment:
Minor, but I'm trying to move towards a style where we explicitly name all our boolean arguments. Something like `{ isRegex: true }` is what I'm thinking here.